pkg/controller/pvcontroller/controller.go

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
I took one layer down so that we can be extended to `n` controllers in the same repo but I don't see in near feature with that requirement so we can remove extra package `pvcontroller`

Reply:
Followed same pattern as per discussion...
```sh
 tree pkg/controller/
pkg/controller/
├── base_controller.go
├── pv_controller.go
├── pv_controller_sync.go
├── pv_controller_sync_test.go
└── types.go

0 directories, 5 files
```